clients/cli/src/main/java/org/apache/gravitino/cli/CommandContext.java

```diff
@@ -55,6 +57,7 @@ public CommandContext(CommandLine line) {
             ? line.getOptionValue(GravitinoOptions.OUTPUT)
             : Command.OUTPUT_FORMAT_PLAIN;
     this.quiet = line.hasOption(GravitinoOptions.QUIET);
+    this.outputLimit = -1;
```

Comment:
It's unclear to me what this is for - perhaps it needs a better name or better still in another PR